File: Problem9.cpp
#include <iostream>
  int main() { //simple func
    int a, b, c, x = 1000; //one Pythagorean triplet for which a + b + c = 1000.
    for (a = 1; a < x; a++) { //a < b < c
      for (b = a; b < x / 2; b++) {
        c = x - a - b; //c is x minus a minus b
        if (a * a + b * b == c * c) { //a^2+b^2=c^2
          std::cout << a * b * c; //multiply em
        }
      }
    }
  }

File: Problem9.java
public class Triplet {
 public static void main(String[] args){
  int a;  //one Pythagorean triplet for which a + b + c = 1000
  int b;
  int c;
  int x = 1000;
  a = 1;
  if (a < x) do {
   b = a;
   if (b < x / 2) do {
    c = x - a - b;
    if (a * a + b * a == c * c) {
     System.out.println("Triplet for which a + b + c = 1000: ");
     System.out.print(a * b * c);
     System.out.println();
     System.out.println("a: " + a);
     System.out.println("b: " + b);
     System.out.println("c: " + c);
    }
    b++;
   } while (b < x / 2);
   a++;
  } while (a < x);
 }
}
